go.mod

```diff
@@ -53,7 +53,7 @@ require (
 	github.com/eapache/go-xerial-snappy v0.0.0-20180814174437-776d5712da21 // indirect
 	github.com/eapache/queue v1.1.0 // indirect
 	github.com/elastic/beats/v7 v7.0.0-alpha2.0.20220328120630-c721abcf0650
```

Comment:
Once the latest csp changes will be merged - needs to replace it with the new `main` commit SHA/release

Reply:
Will do in other commit